conans/client/tools/system_pm.py

```diff
@@ -11,6 +11,8 @@ class SystemPackageTool(object):
     def __init__(self, runner=None, os_info=None, tool=None):
         env_sudo = os.environ.get("CONAN_SYSREQUIRES_SUDO", None)
```

Comment:
If CONAN_SYSREQUIRES_SUDO is defined, it should be prioritized to any other consideration.
I would add `if env_sudo is None and os.name=='posix' and os.geteuid() == '`

Reply:
changed, thx